docs/topics/migrations.txt

```diff
@@ -218,6 +218,38 @@ Apps without migrations must not have relations (``ForeignKey``,
 ``ManyToManyField``, etc.) to apps with migrations. Sometimes it may work, but
 it's not supported.
 
+Swappable dependencies
+======================
+
+A swappable dependency is a feature that allows you to replace or customize
```

Comment:
This entire paragraph document the term _"swappable dependency"_, but not the `swappable_dependency()` function itself, which was the main goal of the ticket.